### ISSUE TITLE
Is weight_norm after conv1d layer in wavenet_block indispensable?

### ISSUE THREAD

**Issue by hwong39** [MU94W/WaveGlow#1]
Nice work! I found that weight_norm layer is not used in your implementation. Have you done an ablation experiment to analyze the effects of weight_norm? Thanks.

**Comment by MU94W** [comment]
I do omit the weight_norm (WN) layer.
WN is designed to accelerate the training process.
[Weight Normalization: A Simple Reparameterization to Accelerate Training of Deep Neural Networks](https://arxiv.org/pdf/1602.07868.pdf)
But based on my past experience, I found that the acceleration gain from WN is limit and the train of WN is unstable.
[Comparison of Batch Normalization and Weight Normalization Algorithms for the Large-scale Image Classification](https://arxiv.org/pdf/1709.08145.pdf)

Nonetheless, I will add the weight norm layer later actually for a FULL reimplementation. :D

**Comment by hwong39** [comment]
@MU94W Thx for explaining it.
